File: get_swagger.py
import requests
import json

def get_swagger():
    url = "https://online-course-marketplace-backend-dc66.onrender.com/swagger/?format=openapi"
    try:
        response = requests.get(url, timeout=30)
        response.raise_for_status()
        print(json.dumps(response.json(), indent=2))
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    get_swagger()

File: get_swagger.js
const axios = require('axios');

async function getSwagger() {
  try {
    const response = await axios.get('https://online-course-marketplace-backend-dc66.onrender.com/swagger/?format=openapi', {
      timeout: 30000,
      headers: {
        'Accept': 'application/json'
      }
    });
    console.log(JSON.stringify(response.data, null, 2));
  } catch (error) {
    if (error.response) {
      console.error('Status:', error.response.status);
      console.error('Data:', error.response.data);
    } else {
      console.error('Error:', error.message);
    }
  }
}

getSwagger();
